packages/api/src/external/hie/get-hie-initiator.ts

```diff
@@ -14,17 +15,59 @@ export type HieInitiator = {
 
 export async function getHieInitiator(
   patient: Pick<Patient, "id" | "cxId">,
+  facilityId: string | undefined
+): Promise<HieInitiator> {
+  const { organization, facilities } = await getPatientWithDependencies(patient);
+
+  const facility = await getPatientsFacility(patient.id, facilities, facilityId);
+
+  if (isItVendor(organization.type)) {
```

Comment:
nit: I keep reading this as "is it vendor" instead of "is IT vendor" - can make this `isHealthcareItVendor` to be explicit